### ISSUE TITLE
No tray notifications

### ISSUE THREAD

**Issue by rulet** [tutao/tutanota#3300]
Tray icon doesn't change the look after windows notification come. And windows notification just disappear immediately so no way to see that new mail came.
This is on latest Windows 10 Pro 21H1 19043.1110

**Comment by johnbotris** [comment]
@rulet If I understand correctly, you mean the notification will show for ~5 seconds and then hide away, but it doesn't go to the action center? This is the behaviour I'm seeing as well.

**Comment by rulet** [comment]
Exactly. And also no sign on tray icon of tutanota app that email came.

**Comment by xAubert** [comment]
Hi, I'm also experiencing  notification issues (also windows 10 pro), but I can't even see any notification.
 When a mail comes in it is like nothing happened at all, which is quite frustrating.
